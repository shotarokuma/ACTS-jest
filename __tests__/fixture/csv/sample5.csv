Column1,Column2,Column3,Column4,Column5,Column6,Column7,Column8,Column9,Column10,Column11,Column12,Column13,Column14,Column15,Column16,Column17,Column18,Column19,Column20
,true,true,,,,,false,,false,,,,false,80,,false,92,,
1,,,,,,true,,true,true,82,,69,true,,40,false,true,,
,,37,false,,false,,21,,76,true,false,,99,,,,,,
73,true,false,false,,20,,43,,,69,true,,true,10,false,,,,22
false,71,,,,false,,,false,50,,true,false,,,,,,,true
77,48,,,true,false,true,false,,13,72,,64,,,,,true,3,
32,false,,34,false,true,,,,53,87,59,73,,85,29,,,true,
,81,true,86,3,,false,,,,,true,,,80,true,true,,,
9,false,false,,,,,,true,87,true,26,,true,,,,,,
,true,36,,,true,true,true,,false,true,,91,true,true,true,,,,false
8,49,28,,,,,5,,,false,,,,58,57,,18,,
true,96,false,,19,,13,false,,true,,false,false,true,14,,93,62,,12
67,23,,81,true,68,true,,false,,,,true,24,45,,30,76,true,
,,,true,77,,,false,,89,true,,,,,14,false,,false,false
,26,,,,88,false,96,26,,false,,,,,true,37,,,
,,90,,false,,,,true,,false,,,true,,76,,,,
,false,true,,true,36,,57,,false,,,false,,,true,,,37,
true,,true,51,75,51,,,42,,80,true,,false,,77,52,51,false,40
73,true,,,true,,false,,4,,false,,48,,37,false,true,true,50,
31,12,39,,,true,71,,,58,8,,true,,,,,82,true,
