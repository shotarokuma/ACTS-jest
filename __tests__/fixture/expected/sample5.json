[
{"Column1":"","Column2":"true","Column3":"true","Column4":"","Column5":"","Column6":"","Column7":"","Column8":"false","Column9":"","Column10":"false","Column11":"","Column12":"","Column13":"","Column14":"false","Column15":"80","Column16":"","Column17":"false","Column18":"92","Column19":"","Column20":""},
{"Column1":"1","Column2":"","Column3":"","Column4":"","Column5":"","Column6":"","Column7":"true","Column8":"","Column9":"true","Column10":"true","Column11":"82","Column12":"","Column13":"69","Column14":"true","Column15":"","Column16":"40","Column17":"false","Column18":"true","Column19":"","Column20":""},
{"Column1":"","Column2":"","Column3":"37","Column4":"false","Column5":"","Column6":"false","Column7":"","Column8":"21","Column9":"","Column10":"76","Column11":"true","Column12":"false","Column13":"","Column14":"99","Column15":"","Column16":"","Column17":"","Column18":"","Column19":"","Column20":""},
{"Column1":"73","Column2":"true","Column3":"false","Column4":"false","Column5":"","Column6":"20","Column7":"","Column8":"43","Column9":"","Column10":"","Column11":"69","Column12":"true","Column13":"","Column14":"true","Column15":"10","Column16":"false","Column17":"","Column18":"","Column19":"","Column20":"22"},
{"Column1":"false","Column2":"71","Column3":"","Column4":"","Column5":"","Column6":"false","Column7":"","Column8":"","Column9":"false","Column10":"50","Column11":"","Column12":"true","Column13":"false","Column14":"","Column15":"","Column16":"","Column17":"","Column18":"","Column19":"","Column20":"true"},
{"Column1":"77","Column2":"48","Column3":"","Column4":"","Column5":"true","Column6":"false","Column7":"true","Column8":"false","Column9":"","Column10":"13","Column11":"72","Column12":"","Column13":"64","Column14":"","Column15":"","Column16":"","Column17":"","Column18":"true","Column19":"3","Column20":""},
{"Column1":"32","Column2":"false","Column3":"","Column4":"34","Column5":"false","Column6":"true","Column7":"","Column8":"","Column9":"","Column10":"53","Column11":"87","Column12":"59","Column13":"73","Column14":"","Column15":"85","Column16":"29","Column17":"","Column18":"","Column19":"true","Column20":""},
{"Column1":"","Column2":"81","Column3":"true","Column4":"86","Column5":"3","Column6":"","Column7":"false","Column8":"","Column9":"","Column10":"","Column11":"","Column12":"true","Column13":"","Column14":"","Column15":"80","Column16":"true","Column17":"true","Column18":"","Column19":"","Column20":""},
{"Column1":"9","Column2":"false","Column3":"false","Column4":"","Column5":"","Column6":"","Column7":"","Column8":"","Column9":"true","Column10":"87","Column11":"true","Column12":"26","Column13":"","Column14":"true","Column15":"","Column16":"","Column17":"","Column18":"","Column19":"","Column20":""},
{"Column1":"","Column2":"true","Column3":"36","Column4":"","Column5":"","Column6":"true","Column7":"true","Column8":"true","Column9":"","Column10":"false","Column11":"true","Column12":"","Column13":"91","Column14":"true","Column15":"true","Column16":"true","Column17":"","Column18":"","Column19":"","Column20":"false"},
{"Column1":"8","Column2":"49","Column3":"28","Column4":"","Column5":"","Column6":"","Column7":"","Column8":"5","Column9":"","Column10":"","Column11":"false","Column12":"","Column13":"","Column14":"","Column15":"58","Column16":"57","Column17":"","Column18":"18","Column19":"","Column20":""},
{"Column1":"true","Column2":"96","Column3":"false","Column4":"","Column5":"19","Column6":"","Column7":"13","Column8":"false","Column9":"","Column10":"true","Column11":"","Column12":"false","Column13":"false","Column14":"true","Column15":"14","Column16":"","Column17":"93","Column18":"62","Column19":"","Column20":"12"},
{"Column1":"67","Column2":"23","Column3":"","Column4":"81","Column5":"true","Column6":"68","Column7":"true","Column8":"","Column9":"false","Column10":"","Column11":"","Column12":"","Column13":"true","Column14":"24","Column15":"45","Column16":"","Column17":"30","Column18":"76","Column19":"true","Column20":""},
{"Column1":"","Column2":"","Column3":"","Column4":"true","Column5":"77","Column6":"","Column7":"","Column8":"false","Column9":"","Column10":"89","Column11":"true","Column12":"","Column13":"","Column14":"","Column15":"","Column16":"14","Column17":"false","Column18":"","Column19":"false","Column20":"false"},
{"Column1":"","Column2":"26","Column3":"","Column4":"","Column5":"","Column6":"88","Column7":"false","Column8":"96","Column9":"26","Column10":"","Column11":"false","Column12":"","Column13":"","Column14":"","Column15":"","Column16":"true","Column17":"37","Column18":"","Column19":"","Column20":""},
{"Column1":"","Column2":"","Column3":"90","Column4":"","Column5":"false","Column6":"","Column7":"","Column8":"","Column9":"true","Column10":"","Column11":"false","Column12":"","Column13":"","Column14":"true","Column15":"","Column16":"76","Column17":"","Column18":"","Column19":"","Column20":""},
{"Column1":"","Column2":"false","Column3":"true","Column4":"","Column5":"true","Column6":"36","Column7":"","Column8":"57","Column9":"","Column10":"false","Column11":"","Column12":"","Column13":"false","Column14":"","Column15":"","Column16":"true","Column17":"","Column18":"","Column19":"37","Column20":""},
{"Column1":"true","Column2":"","Column3":"true","Column4":"51","Column5":"75","Column6":"51","Column7":"","Column8":"","Column9":"42","Column10":"","Column11":"80","Column12":"true","Column13":"","Column14":"false","Column15":"","Column16":"77","Column17":"52","Column18":"51","Column19":"false","Column20":"40"},
{"Column1":"73","Column2":"true","Column3":"","Column4":"","Column5":"true","Column6":"","Column7":"false","Column8":"","Column9":"4","Column10":"","Column11":"false","Column12":"","Column13":"48","Column14":"","Column15":"37","Column16":"false","Column17":"true","Column18":"true","Column19":"50","Column20":""},
{"Column1":"31","Column2":"12","Column3":"39","Column4":"","Column5":"","Column6":"true","Column7":"71","Column8":"","Column9":"","Column10":"58","Column11":"8","Column12":"","Column13":"true","Column14":"","Column15":"","Column16":"","Column17":"","Column18":"82","Column19":"true","Column20":""}
]
